src/data/general-cs-quiz.ts

```diff
@@ -1457,6 +1457,15 @@ const generalCSQuiz = [
     Distractor3: "number*of*donuts",
     Explanation: "Kebab case separates each lowercase word by dashes.",
     Link: "https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/"
+  },
+  {
+    Question: "Which of the following type of tree is also known as 'large key' tree?",
+    Answer: "B-Tree",
+    Distractor1: "Binary Search Tree",
+    Distractor2: "Adelson-Velsky and Landis Tree (AVL Tree/Height balanced tree)",
+    Distractor3: "Ternary Tree",
+    Explanation: "B-Trees are characterized by the large number of keys that they can store in a single node, which is why they are also known as large key trees",
```

Comment:
It looks like this explanation was lifted from Geeks for Geeks
https://www.geeksforgeeks.org/introduction-of-b-tree-2/

We want to avoid that since that is plagiarism

```suggestion
    Explanation: "B-Trees are identified by their capacity to hold a significant quantity of keys within a single node, leading to their alternative name, large key trees.",
```